### PR TITLE
cli: update force_new_epoch() and add set_config() to NeofsAdm

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: src/neofs_testlib/cli/neofs_adm/morph.py [hunk 1]
```diff
@@ -354,3 +354,23 @@ def update_contracts(
                 if param not in ["self"]
             },
         )
+
+    def set_config(self, rpc_endpoint: str, alphabet_wallets: str, post_data: str) -> CommandResult:
+        """Create new NeoFS epoch event in the side chain.
```
Comment (roman-khimov) [inline src/neofs_testlib/cli/neofs_adm/morph.py:359]: What?

File: src/neofs_testlib/cli/neofs_adm/morph.py [hunk 2]
```diff
@@ -127,7 +127,7 @@ def dump_hashes(self, rpc_endpoint: str) -> CommandResult:
         )
 
     def force_new_epoch(
-        self, rpc_endpoint: Optional[str] = None, alphabet: Optional[str] = None
+        self, rpc_endpoint: Optional[str] = None, alphabet_wallets: Optional[str] = None
```
Comment (roman-khimov) [inline src/neofs_testlib/cli/neofs_adm/morph.py:130]: I'd split it into a separate commit, since it's a different (from set_config()) problem.

File: src/neofs_testlib/cli/neofs_adm/morph.py (hunk 1 above)
Comment (evgeniiz321) [reply src/neofs_testlib/cli/neofs_adm/morph.py:359]: oops, ctrl+c/ctrl+v didn't work well

File: src/neofs_testlib/cli/neofs_adm/morph.py (hunk 2 above)
Comment (evgeniiz321) [reply src/neofs_testlib/cli/neofs_adm/morph.py:130]: done

File: src/neofs_testlib/cli/neofs_adm/morph.py (hunk 1 above)
Comment (evgeniiz321) [reply src/neofs_testlib/cli/neofs_adm/morph.py]: fixed

**Review by roman-khimov — APPROVED** [review]
